### PR TITLE
Addition of CSS Bitcoin People.

### PR DESCRIPTION
**Pull request by davexbtc**
Addition of CSS Bitcoin People.
Bitcoin People is an italian company focused on Bitcoin.
This is our customization.

### CODE REVIEW

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
You didn't add an image?

**Comment by davecl16** [comment]
Sorry, this is the image: https://mlcqxhkwihu1.i.optimole.com/yaRM1Ss-csyJyoUs/w:478/h:74/q:90/https://www.bitcoinpeople.it/wp-content/uploads/2020/08/cropped-Bitcoin-People-logo.png
Can you add it? Thanks

**Comment by davecl16** [comment]
Hi, any update?

**Comment by davexbtc** [comment]
Hi @pavlenex , any update? Thanks

File: docs/Development/Theme.md [hunk 1]
```diff
@@ -2027,3 +2027,259 @@ CSS file:
 .action-button:active:hover, .action-button:active:focus, .action-button:active.focus, .action-button.active:hover, .action-button.active:focus, .action-button.active.focus, .open>.action-button.dropdown-toggle:hover, .open>.action-button.dropdown-toggle:focus, .open>.action-button.dropdown-toggle.focus {    color: #fff;    background-color: #3aa821;    border-color: #aa0303;}
 ```
 </details>
+
+---
+
+### [Bitcoin People](https://bitcoinpeople.it)
+
+![Bitcoin People checkout](../img/Theme-BitcoinPeople.png)
```
Comment (britttttk) [inline docs/Development/Theme.md:2035]: @davexbtc you need to add the image to your pull request, as Pavlenex already mentioned.

You can see your image is broken in the document preview: https://github.com/btcpayserver/btcpayserver-doc/blob/90a3fad76a436a4c5b7ff382a74aea5d710feacf/docs/Development/Theme.md#bitcoin-people

Let me know if you don't know how to do that and I can explain further. Your Pull Request will not be merged until you complete this.

**Review by britttttk — CHANGES_REQUESTED** [review]

File: docs/Development/Theme.md (hunk 1 above)
Comment (davexbtc) [reply docs/Development/Theme.md:2035]: I think I've done.

Comment (davexbtc) [reply docs/Development/Theme.md:2035]: I made the pull request 1014.

**Review by dennisreimann — APPROVED** [review]

File: docs/Development/Theme.md (hunk 1 above)
Comment (dennisreimann) [reply docs/Development/Theme.md:2035]: Added it here.

**Pull request merged**